src/docs/cookbook/animation/page-route-animation.md

```diff
@@ -27,7 +25,7 @@ To create a custom page route transition, this recipe uses the following steps:
 4. Use a `CurveTween`
 5. Combine the two `Tween`s
 
-# 1. Set up a PageRouteBuilder
+## 1. Set up a PageRouteBuilder
```

Comment:
Thanks for taking care of this! I noticed it after we published your last page, but hadn't gotten around to fixing it.